src/signals/IncidentMap/components/IncidentLayer.tsx

```diff
@@ -27,39 +28,19 @@ type Properties = {
   created_at: string
 }
 
-type Feature = GeoJSONFeature<Point, Properties>
+/* istanbul ignore next */
+const clusterLayerOptions = {
+  zoomToBoundsOnClick: true,
+  chunkedLoading: true,
+}
 
 const IncidentLayer = () => {
   const [mapMessage, setMapMessage] = useState<ReactElement | string>()
+  const [layerInstance, setLayerInstance] = useState<L.LayerGroup>()
```

Comment:
Volgens mij is dit de type: `L.GeoJSON<Point>`. Dan kun je de typecast op line 87 weghalen.

Reply:
mooi!